client/index.jsx

```diff
@@ -14,8 +22,8 @@ class Pictures extends React.Component {
   }
 
   componentDidMount() {
-    const { itemId } = this.props;
-    axios.get('/pictures', {
+    const itemId = window.location.pathname.slice(1);
+    axios.get('http://localhost:3000/pictures', {
```

Comment:
Suggestion to separate your server url into a variable so it will be easier to change later in case you have more than one request from your frontend.